src/ui/buttons/buttons.jsx

```diff
@@ -203,6 +203,12 @@ export function Buttons(props : ButtonsProps) : ElementNode {
                     : null
             }
 
+            {
+                (fundingSources.indexOf(FUNDING.EPS) !== -1)
```

Comment:
Should we add a condition that will incorporate any future funding sources? Currently, this is EPS specific